docs/docs/use_cases/tool_use/agents.ipynb

```diff
@@ -173,27 +170,45 @@
    "id": "85e9875a-d8d4-4712-b3f0-b513c684451b",
    "metadata": {},
    "source": [
-    "## Create agent"
+    "## Create agent\n",
+    "\n",
+    "We'll need to use a model with tool calling capabilities. You can see which models support tool calling [here](/docs/integrations/chat/).\n",
+    "\n",
+    "```{=mdx}\n",
+    "import ChatModelTabs from \"@theme/ChatModelTabs\";\n",
+    "\n",
+    "<ChatModelTabs customVarName=\"llm\"/>\n",
```

Comment:
we might `hideCohere` pending https://github.com/langchain-ai/langchain-cohere/pull/15